### PR TITLE
Add scan-pull-requests command for GitHub Actions

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
---

### CODE REVIEW

File: templates/github-actions/README.md [hunk 1]
```diff
@@ -41,3 +41,18 @@ above. This will make Frogbot open pull requests with fixes for security vulnera
...
+### Scan Open Pull Requests for Multiple Repositories
+
+Create a new repository named `Frogbot`.
+
+Make a folder named `.jfrog` under the newly created repository and add a `frogbot-config.yml` file to it, following the
+template for `frogbot-config.yml`.
+
+Create a workflow file named `frogbot-scan-pull-requests.yml` under the `.github/workflows` folder in the `Frogbot`
+repository. Depending on the tool used to download the project dependencies, uncomment the install prerequisites inside
+the workflow file.
```
Comment (yahavi) [inline templates/github-actions/README.md:54]: Let's add a link to the [frogbot-scan-pull-requests.yml](scan-pull-requests/frogbot-scan-pull-requests.yml) file:
```
[frogbot-scan-pull-requests.yml](scan-pull-requests/frogbot-scan-pull-requests.yml)
```

File: templates/github-actions/README.md [hunk 2]
```diff
@@ -41,3 +41,18 @@ above. This will make Frogbot open pull requests with fixes for security vulnera
...
+
+Make sure to follow the guidelines in the 'Using the Templates' section above.
```
Comment (yahavi) [inline templates/github-actions/README.md:56]: I'm not sure that the instructions in `Using the Templates` are relevant here.
```suggestion
```

File: templates/github-actions/README.md [hunk 3]
```diff
@@ -41,3 +41,18 @@ above. This will make Frogbot open pull requests with fixes for security vulnera
 - [Pip](scan-and-fix/frogbot-scan-and-fix-pip.yml)
 - [Yarn 2](scan-and-fix/frogbot-scan-and-fix-yarn.yml)
 - [Pipenv](scan-and-fix/frogbot-scan-and-fix-pipenv.yml)
+
+### Scan Open Pull Requests for Multiple Repositories
```
Comment (yahavi) [inline templates/github-actions/README.md:45]: Let's add a similar title for single repositories. To ensure a strict separation, we may add a link at the top of this README.

File: templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          # JFrog username with 'read' permissions for Xray. Must be provided with JF_PASSWORD
+          JF_USER: ${{ secrets.JF_USER }}
+
+          # [Mandatory if JF_ACCESS_TOKEN is not provided]
+          # JFrog password. Must be provided with JF_USER
+          JF_PASSWORD: ${{ secrets.JF_PASSWORD }}
+
+          # [Mandatory]
+          # GitHub Personal Access Token (PAT)
+          JF_GIT_TOKEN: ${{ secrets.GH_PAT }}
```
Comment (yahavi) [inline templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml:67]: Shouldn't be ```${{ secrets.GITHUB_TOKEN }```?

File: templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+      # - name: Set up Java
+      #   uses: actions/setup-java@v3
+      #   with:
+      #     java-version: "11"
+      #     distribution: "temurin"
+
+      # Uncomment if you have Gradle project
+      # - name: Setup Gradle
+      #   uses: gradle/gradle-build-action@v2
+
```
Comment (yahavi) [inline templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml:50]: I think that Go, npm, and Yarn are missing

File: templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+      - uses: actions/checkout@v2
+
+      # Install prerequisites
+
+      # Uncomment if you have a Pip/Pipenv/Poetry project
+      # - uses: actions/setup-python@v4
+      #   with:
+      #     python-version: "3.x"
+
+      # Uncomment if you have Pipenv project
```
Comment (yahavi) [inline templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml:22]: (same for the others)
```suggestion
      # Uncomment if you have a Pipenv project
```

File: templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    - cron: '*/5 * * * *'
+permissions:
+  pull-requests: write
+  contents: write
+jobs:
+  scan-pull-requests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (yahavi) [inline templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml:13]:
```suggestion
      - uses: actions/checkout@v3
```

**Review by yahavi — CHANGES_REQUESTED** [review]

File: templates/github-actions/README.md (hunk 1 above)
Comment (omerzi) [reply templates/github-actions/README.md:54]: done

File: templates/github-actions/README.md (hunk 2 above)
Comment (omerzi) [reply templates/github-actions/README.md:56]: done

File: templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml (hunk 6 above)
Comment (omerzi) [reply templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml:22]: done

File: templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml (hunk 5 above)
Comment (omerzi) [reply templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml:50]: done

File: templates/github-actions/README.md (hunk 3 above)
Comment (omerzi) [reply templates/github-actions/README.md:45]: done

**Review by eyalbe4 — APPROVED** [review]
PR approved, but we'll need to create a follow-up PR to restructure the documentation.

**Pull request merged**